source/common/grpc/codec.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <array>
+
+#include "envoy/buffer/buffer.h"
+
+namespace Grpc {
+// Last bit for an expanded message without compression.
+const uint8_t GRPC_FH_DEFAULT = 0b0u;
+// Last bit for a compressed message.
+const uint8_t GRPC_FH_COMPRESSED = 0b1u;
+
+enum class CompressionAlgorithm { None, Gzip };
+
+struct Frame {
+  uint8_t flags;
```

Comment:
We have been doing this for public structs so to be consistent I would change it. (I don't really care that much about this but might as well be consistent).